### PR TITLE
token: add @FILE_LINE

### PR DESCRIPTION
**Pull request by seven1240**
add `@FILE_LINE` which is like `@FILE:@LINE` makes it easy to write logs.

### CODE REVIEW

File: doc/docs.md [hunk 1]
```diff
@@ -5580,8 +5580,9 @@ that are substituted at compile time:
 - `@METHOD` => replaced with ReceiverType.MethodName
 - `@MOD` => replaced with the name of the current V module
 - `@STRUCT` => replaced with the name of the current V struct
-- `@FILE` => replaced with the path of the V source file
+- `@FILE` => replaced with the absolute path of the V source file
 - `@LINE` => replaced with the V line number where it appears (as a string).
+- `@FILE_LINE` => like `@FILE:@LINE`, but FILE is in relative path
```
Comment (spytheman) [inline doc/docs.md:5585]:
```suggestion
- `@FILE_LINE` => like `@FILE:@LINE`, but the file part is a relative path 
```

**Comment by spytheman** [comment]
You have to run `v fmt -w vlib/v/checker/checker.v` .

**Comment by seven1240** [comment]
Thank you. I updated. but I'm not sure why the first one works but not the second one. the line number of line2 always equals line1 + 1 in the second case.

```
	line1, line2 := '${@LINE}', '${@FILE_LINE}'
	line1, line2 := ${@LINE}, ${@FILE_LINE}
```

**Comment by JalonSolov** [comment]
The 2nd line doesn't do string interpolation, since there are no quotes.  I'd actually expect that line to be an error.

**Comment by seven1240** [comment]
sorry, the second like was `line1, line2 := @LINE, @FILE_LINE`

**Comment by spytheman** [comment]
> sorry, the second line was `line1, line2 := @LINE, @FILE_LINE`

That is an unrelated bug - the line number of the node position is wrong, for the last token on the line of a multi assignment. Good find.

For example:
```v
line1, line2, line3 := @LINE, @LINE, @LINE
dump(line1)
dump(line2)
dump(line3)
```

**Comment by spytheman** [comment]
I've filed last token line bug in <https://github.com/vlang/v/issues/15672>.

**Pull request merged**